ui/app/manager/src/pages/page-alarms.ts

```diff
@@ -0,0 +1,1133 @@
+import {css, html, PropertyValues, TemplateResult, unsafeCSS} from "lit";
+import {customElement, property, query, state} from "lit/decorators.js";
+import {AppStateKeyed, Page, PageProvider, router} from "@openremote/or-app";
+import {Store} from "@reduxjs/toolkit";
+import {
+    AlarmAssetLink,
+    AlarmSeverity,
+    AlarmSource,
+    AlarmStatus,
+    AlarmUserLink,
+    Asset,
+    SentAlarm,
+    User,
+    UserQuery
+} from "@openremote/model";
+import manager, {DefaultColor3, DefaultColor4} from "@openremote/core";
+import i18next from "i18next";
+import {showSnackbar} from "@openremote/or-mwc-components/or-mwc-snackbar";
+import {GenericAxiosResponse, isAxiosError} from "@openremote/rest";
+import {getAlarmsRoute} from "../routes";
+import {when} from "lit/directives/when.js";
+import {until} from "lit/directives/until.js";
+import {InputType, OrInputChangedEvent, OrMwcInput} from "@openremote/or-mwc-components/or-mwc-input";
+import {OrMwcDialog, showDialog, showOkCancelDialog} from "@openremote/or-mwc-components/or-mwc-dialog";
+import {OrAssetTreeRequestSelectionEvent, OrAssetTreeSelectionEvent} from "@openremote/or-asset-tree";
+import {OrMwcTable, OrMwcTableRowClickEvent, TableColumn, TableRow} from "@openremote/or-mwc-components/or-mwc-table";
+import {MDCDataTable} from "@material/data-table";
+
+const tableStyle = require("@material/data-table/dist/mdc.data-table.css");
```

Comment:
Are these imports necessary for `page-alarms`? I would expect it in `or-mwc-table` but not here. If it's regards in styling, try adding `import "@openremote/or-mwc-components/or-mwc-table";` alongside of the current import.